indexer/src/main/java/org/uniprot/store/indexer/keyword/KeywordStatisticsStep.java

```diff
@@ -77,6 +76,6 @@ public Step keywordStatistics(
     }
 
     protected String getStatisticsSQL() {
```

Comment:
I think this method is not necessary anymore, because we do not have the Fake implementation and we do not need to override the SQL for tests. You could use directly the constant in itemKeywordStatisticsReader method